### DISCUSSION TITLE
Image Title Removal

### DISCUSSION THREAD

**Discussion by nathanstreetx** [stashapp/stash#5409] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
I would like to be able to remove the titles from the individual images within a gallery, as they are either repetitive titles or just a random series of numbers, and are not necessary for me. 

**Describe alternatives you've considered**
I had a similar request regarding the description display that was solved with a CSS Script (#5346). I attempted to modify the scripts I was provided then, but being unfamiliar with the programming language left me unsuccessful. User @bob123491234 was extremely helpful with my issue last time.

**Additional context**
Current View within a Gallery
![image](https://github.com/user-attachments/assets/010f65ea-0966-4ed4-a806-a7aed60ac54e)

Desired View
![image](https://github.com/user-attachments/assets/40df4a67-c8cf-445b-aa07-3dbf7d17ca5c)

**Comment by echo6ix** [comment, marked as answer]
Add this Custom CSS and refresh Stash.

```css
/* Hide image titles in image cards */
.image-card {
  .card-section-title {
    display: none;
  }
}
```

**Reply by nathanstreetx** [reply]
That worked a treat. Thanks!
